Added Building Web APIs with ASP.NET Core

stjepanjurekovic opened the pull request:
CATEGORY/PROJECT_NAME - [PROJECT_LINK](PROJECT_LINK)

Hi,

I've added this title to your list because I believe it's a good match. Thank you for considering it.

Best,

<!--
  Please, fill in this PR template. It will help maintainers to do the review.
  Also, please read our [Contribution guidelines](CONTRIBUTING.md). 
  Please, create one pull request per link.
-->

stjepanjurekovic commented:
Thank you.

Best,

On Wed, Oct 5, 2022 at 1:27 AM Wade Baglin ***@***.***> wrote:

> Merged #1117 <https://github.com/quozd/awesome-dotnet/pull/1117> into
> master.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quozd/awesome-dotnet/pull/1117#event-7520689694>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANDIEMP62ZBVLNWP7YKXDS3WBS4MZANCNFSM6AAAAAAQYRHBAA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Stjepan Jureković
Manning Publications - Marketing
***@***.***

amalsharief commented:
Hi, 
Where is the project link?

stjepanjurekovic commented:
Hi,

Can the project link be a link to the book in our catalog?
If so, here it is:
https://www.manning.com/books/asp-net-core-in-action-third-edition

Best,

On Tue, Oct 11, 2022 at 10:05 AM amalsharief ***@***.***>
wrote:

> Hi,
> Where is the project link?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quozd/awesome-dotnet/pull/1117#issuecomment-1274266380>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANDIEMNRQBUPAZKWMZWGSI3WCUNV7ANCNFSM6AAAAAAQYRHBAA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Stjepan Jureković
Manning Publications - Marketing
***@***.***

pleb commented:
@stjepanjurekovic The list already points to this edition. Am I missing something?

![image](https://user-images.githubusercontent.com/595869/195214471-4687bc6c-d6e2-4522-8dde-0b1a95cf9244.png)

stjepanjurekovic commented:
Hi Wade,

The title is for the first edition, while the link is for the third
edition. When I saw the title and automatically thought it should be for
the first edition and that my predecessor left it here. Now, that you've
point it out, I am confused, too.

With your approval, I can edit both: add the link to the first edition to
this entry, and make sure the new entry links to the third edition.

Best,

On Wed, Oct 12, 2022 at 1:09 AM Wade Baglin ***@***.***>
wrote:

> @stjepanjurekovic <https://github.com/stjepanjurekovic> The list already
> points to this edition. Am I missing something?
>
> [image: image]
> <https://user-images.githubusercontent.com/595869/195214471-4687bc6c-d6e2-4522-8dde-0b1a95cf9244.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quozd/awesome-dotnet/pull/1117#issuecomment-1275387351>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANDIEMJG6XZDU3JRF22VKKTWCXXTVANCNFSM6AAAAAAQYRHBAA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Stjepan Jureković
Manning Publications - Marketing
***@***.***

pleb commented:
@stjepanjurekovic I've updated the book section and added the additions for one

stjepanjurekovic commented:
Awesome, thanks!

Best,

On Mon, Oct 17, 2022 at 1:26 AM Wade Baglin ***@***.***>
wrote:

> @stjepanjurekovic <https://github.com/stjepanjurekovic> I've updated the
> book section and added the additions for one
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quozd/awesome-dotnet/pull/1117#issuecomment-1280092491>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANDIEMIW6LUZ6VTGK4L22DLWDSFKTANCNFSM6AAAAAAQYRHBAA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Stjepan Jureković
Manning Publications - Marketing
***@***.***
